tests/test_misc.py

```diff
@@ -1927,7 +1927,7 @@ def test_large_spends(revault_network, bitcoind, executor):
     revault_network.activate_fresh_vaults(vaults)
 
     feerate = 1
-    n_outputs = random.randint(1, 5)
+    n_outputs = random.randint(1, 3)
```

Comment:
Huh, how comes? It's a pretty explicit showcase of the computation being wrong since it could get announced succesfully..

Reply:
And it's actually not needed, test pass with:
```diff
diff --git a/tests/servers/coordinatord b/tests/servers/coordinatord
--- a/tests/servers/coordinatord
+++ b/tests/servers/coordinatord
@@ -1 +1 @@
-Subproject commit 8178de74ca7d2958b13f208db60d55a3d3ebf2d9
+Subproject commit 8178de74ca7d2958b13f208db60d55a3d3ebf2d9-dirty
diff --git a/tests/test_misc.py b/tests/test_misc.py
index 3df6aa1..55a4b7f 100644
--- a/tests/test_misc.py
+++ b/tests/test_misc.py
@@ -1927,7 +1927,7 @@ def test_large_spends(revault_network, bitcoind, executor):
     revault_network.activate_fresh_vaults(vaults)
 
     feerate = 1
-    n_outputs = random.randint(1, 5)
+    n_outputs = 5
     fees = revault_network.compute_spendtx_fees(feerate, len(deposits), n_outputs)
     destinations = {
         bitcoind.rpc.getnewaddress(): (total_amount - fees) // n_outputs
```
So must be an artifact i guess

Reply:
I mean, the computation is not perfect, and it might have been that the test was a bit flaky when n_outputs=5...
But yeah I can check the size of the final tx when outputs=5 and compare with the estimation :)

Reply:
I just checked, with 5 outputs the algorithm says the spend will be 65590 bytes (too big), while the actual spend turns out to be 65142. Again, the calculation can't be perfect, I think it's good enough :)

Reply:
I messed with my branch, you are right